treeherder/webapp/api/serializers.py

```diff
@@ -97,6 +97,37 @@ class Meta:
         fields = '__all__'
 
 
+class JobSerializer(serializers.ModelSerializer):
+
+    def to_representation(self, job):
+        option_collection_map = self.context['option_collection_map']
+
+        return [
+            models.Job.get_duration(
+                job['submit_time'], job['start_time'], job['end_time']
+            ),
+            job['failure_classification_id'],
```

Comment:
I'm curious why these are all being returned from this method. Are they all being transformed in some way?

Reply:
I think you can just return the `models.Job.get_duration` here and then add this JobSerializer as a field to your main serializer where you return all fields.

Reply:
@sarah-clements and I discussed this in Zoom yesterday, but this is because we are returning an array of values only, not a dictionary for each job.  We discussed a few options, but in the end, I stuck with this serializer.  I was, however, able to reduce the code here by using ``job.values()``.